templates/core/unauthorized.json.twig

```diff
@@ -0,0 +1,6 @@
+{
+    "error": {
+        "location": "{{ pageTitle|attr_js }}",
+        "message": "{{ "Not Authorized"|xlt }}"
+    }
+}
```

Comment:
goodness. i am at a loss of how these json things should be escaped :) will need to analyze the code where these are used a bit more.

Reply:
can't find use cases for the json. looks like they all go through validate_messages_document_ajax.php script is user sets the $_GET['format'] to json. And doesn't look like any way to sneak a variable into these via a request. So for now would escape these all via json_encode. So would consider doing:
```
        "location": {{ pageTitle|json_encode }},
        "message": {{ "Not Authorized"|xl| json_encode }}
```

Reply:
ok

Reply:
fixed

Reply:
need to remove the surrounding double quotes

Reply:
Fixed again!  haha